Examples/Detectors/DD4hepDetector/compact/Calorimeter/Calorimeter_LAR.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lccdd xmlns:compact="http://www.lcsim.org/schemas/compact/1.0"
+    xmlns:xs="http://www.w3.org/2001/XMLSchema"
+    xs:noNamespaceSchemaLocation="http://www.lcsim.org/schemas/compact/1.0/compact.xsd">
+
+    <includes>
+        <gdmlFile ref="../elements.xml"/>
+        <gdmlFile ref="../materials.xml"/>
+    </includes>
+
+    <info name="Test-Calorimeter"
+        title="Test-Calorimeter"
+        author="corentin.allaire@cern.ch" url="" status="test" version="0">
+        <comment>Calorimeter test for the mapping and the propagation</comment>
+    </info>
+    <define>
+        <!--World-->
+        <constant name="world_size" value="10.*m"/>
+        <constant name="world_x" value="world_size"/>
+        <constant name="world_y" value="world_size"/>
+        <constant name="world_z" value="world_size"/>
+        <!--Dimensions of the beam pipe -->
```

Comment:
Is there a beam pipe required?

Reply:
The beampipe is not required. From the point of view of mapping test it is nice since it allows us to test both surface and volume mapping on the same geometry.

Reply:
I thought that there might be a beampipe.xml but ok, fine

Reply:
No I just copied the implementation from the ODD...